mozetl/search/daily_search_rollups.py

```diff
@@ -72,6 +77,7 @@ def add_profile_day(frame):
   {}
 WHERE
   ((search_count > -1) OR (search_count is null))
+  AND search_source IN ('searchbar', 'urlbar', 'abouthome', 'newtab', 'contextmenu', 'system')
```

Comment:
I think we also need `'NO_SEARCHES'` in here to pull in the rows from the `unwrapped_nulls` table.